app/models/applicant.rb

```
@@ -1,2 +1,9 @@
 class Applicant < ApplicationRecord
+  @@CASEID_ENCRYPTION_KEY = Base64.decode64 ENV.fetch 'CASEID_ENCRYPTION_KEY'
+  @@CELLPHONENUMBER_ENCRYPTION_KEY = Base64.decode64 ENV.fetch 'CELLPHONENUMBER_ENCRYPTION_KEY'
```

I like how you use `fetch`, which will cause missing envs to fail loudly.